Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -604,6 +605,8 @@ static Ref<Inspector::Protocol::Page::Cookie> buildObjectForCookie(const Cookie&
         .setSecure(cookie.secure)
         .setSameSite(cookieSameSitePolicyJSON(cookie.sameSite))
         .release();
+    inspectorCookie->setPartitionKey(cookie.partitionKey);
```

Comment:
will the `partitionKey` always exist?  or is it sometimes `""`/`null`?  if the latter, id suggest we avoid setting it unless it's actually valid

Reply:
It is sometimes null.